ts/screens/portfolio/AddCardScreen.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+    Button,
+    Col,
+    Content,
+    Grid,
+    Icon,
+    Input,
+    Item,
+    Text
+} from "native-base";
+import * as React from "react";
+import {
+  FlatList,
+  Image,
+  View
+} from "react-native";
+import { NavigationScreenProp, NavigationState} from "react-navigation";
+
+import I18n from "../../i18n"
+import ROUTES from "../../navigation/routes"
+
+type Props = {
+    navigation: NavigationScreenProp<NavigationState>
+}
+
+type State = {
+    cardNumber: string,
+    expireDate: string,
+    secureCode: string,
+    cardHolder: string
+}
+
+export class AddCardScreen extends React.Component<Props, State> {
+
+  public static navigationOptions = {
+      title: I18n.t("portfolio.addCardTitle"),
+      headerBackTitle: null
+  }
+
+  constructor(props: Props) {
+    super(props)
+    this.state = {
+        cardNumber: "",
+        expireDate: "",
+        secureCode: "",
+        cardHolder: ""
+    }
+  }
+
+  public render(): React.ReactNode {
+
+    const cardItems = [
```

Comment:
perhaps can be refactored with the list from `CreditCardComponent.tsx` ? why maintain two maps of card types to images?